config/parser.go

```diff
@@ -102,13 +103,35 @@ type Defaults struct {
 }
 
 type Keybinding struct {
-	Key     string `yaml:"key"`
-	Command string `yaml:"command"`
+	Key     string   `yaml:"key"`
+	Keys    []string `yaml:"keys"`
+	Command string   `yaml:"command"`
+	Builtin string   `yaml:"builtin"`
+}
+
+func (kb Keybinding) NewBinding(previous *key.Binding) key.Binding {
+	helpDesc := ""
+	if previous != nil {
+		helpDesc = previous.Help().Desc
+	}
+
+	if len(kb.Keys) > 0 {
```

Comment:
Do you expect people to want alts? Is this a personal need of yours? I don't expect many users will actually define multiple keys for the same action.
If there's not a big use case, maybe it's better to omit to reduce the config bloat

Reply:
Yea I wasn't entirely sure, I can't see myself particularly using it. So that is fair.

Reply:
Removed in latest commit